app/src/main/java/org/mozilla/FirefoxLite/firefoxdownloader/MainActivity.java

```diff
@@ -0,0 +1,294 @@
+import android.content.BroadcastReceiver;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.SharedPreferences;
+import android.content.pm.PackageManager;
+import android.graphics.Color;
+import android.os.Build;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+import android.widget.TimePicker;
+import android.widget.Toast;
+
+
+import java.util.Calendar;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.app.ActivityCompat;
+import androidx.core.app.NotificationCompat;
+import androidx.core.content.ContextCompat;
+import androidx.fragment.app.DialogFragment;
+import androidx.lifecycle.LifecycleOwner;
+
+public class MainActivity extends AppCompatActivity implements TimePickerDialog.OnTimeSetListener {
+
+
+    final String URL1 = "http://mattmahoney.net/dc/enwik8.pmd";
+    final String URL2 = "http://mattmahoney.net/dc/enwik9.pmd";
+    final String URL3 = "http://www.appsapk.com/downloading/latest/WeChat-6.5.7.apk";
+    final String URL4 = "http://www.appsapk.com/downloading/latest/Emoji%20Flashlight%20-%20Brightest%20Flashlight%202018-2.0.1.apk";
+    final String URL5 = "http://www.appsapk.com/downloading/latest/Screen%20Recorder-7.7.apk";
+    final String URL6 = "http://www.appsapk.com/downloading/latest/Call%20Recorder%20-%20Automatic%20Call%20Recorder-1.6.0.apk";
+    final String URL7 = "http://www.appsapk.com/downloading/latest/Sound%20Profile%20(+%20volume%20scheduler)-5.25.apk";
+    final String URL8 = "http://www.appsapk.com/downloading/latest/Evernote%20-%20stay%20organized.-7.9.7.apk";
+    final String URL9 = "http://www.appsapk.com/downloading/latest/UC-Browser.apk";
+    final String URL10 = "http://www.appsapk.com/downloading/latest/Barcode%20Scanner-1.2.apk";
+    final String URL11 = "http://download.blender.org/peach/bigbuckbunny_movies/BigBuckBunny_640x360.m4v";
+    final String URL12 = "http://www2.sdfi.edu.cn/netclass/jiaoan/englit/download/Harry%20Potter%20and%20the%20Sorcerer's%20Stone.pdf";
+    final String URL13 = "https://media.giphy.com/media/Bk0CW5frw4qfS/giphy.gif";
+    final String URL14 = "http://techslides.com/demos/sample-videos/small.mp4";
+    final String URL15 = "http://www.sample-videos.com/video/mp4/720/big_buck_bunny_720p_10mb.mp4";
+
+    Button buttonOne;
+    Button buttonCancelOne;
+    Button buttonScheduleOne;
+
+
+
+
+    TextView textViewProgressOne,textViewProgressTwo,textViewProgressThree, textViewProgressFour, textViewProgressFive, textViewProgressSix,
+            textViewProgressSeven, textViewProgressEight, textViewProgressNine,
```

Comment:
Please use a RecyclerView to display the list. 
I also wonder, how can the client know the current downloading task in the queue?
I think that's also one requirement for this library.